mealie/routes/users/crud.py

```diff
@@ -56,6 +58,18 @@ def delete_user(self, item_id: UUID4):
 
 @controller(user_router)
 class UserController(BaseUserController):
+    @user_router.get("/group-users", response_model=UserPagination)
+    def get_all_group_users(self, q: PaginationQuery = Depends(PaginationQuery)):
+        """Returns all users from the current group"""
+
+        response = self.repos.users.by_group(self.group_id).page_all(
+            pagination=q,
+            override=UserOut,
```

Comment:
Should we use a different type here instead of UserOut? It contains many fields that may be sensitive?

Reply:
Yeah that makes sense, I can add something like UserSummary